examples/run/cpu/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ traccc_add_executable( truth_finding_example "truth_finding_example.cpp"
    LINK_LIBRARIES vecmem::core detray::utils traccc::core traccc::io 
    traccc::performance traccc::options)
 
+traccc_add_executable( truth_fitting_example "truth_fitting_example.cpp"
+   LINK_LIBRARIES vecmem::core detray::utils traccc::core traccc::io 
```

Comment:
Maybe also link against ```detray::io``` explicitly (I guess it part of ```traccc::io```?)

Reply:
Added now